app/utils/command.py

```diff
@@ -0,0 +1,81 @@
+import inspect
+import logging
+from typing import Any
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandParser
+
+
+class CustomBaseCommand(BaseCommand):
```

Comment:
This is taken from service-stac, correct?

Reply:
Yes. It's also the same thing introduced in #23.

Reply:
:+1: